int16 taskID
int16 robotID
int16 action
int16 status